client/app/utils/server.py

```diff
@@ -0,0 +1,23 @@
+"""This is dummy server that is used when developing New API for client."""
+
+import asyncio
+import random
+
+import websockets
+from loguru import logger
+
+
+async def echo(websocket):
+    """Send Random message back."""
+    async for message in websocket:
+        logger.debug(f"Received {message}")
+        await websocket.send(str(random.random()))
+
+
+async def main():
+    """Create and Host the server"""
+    async with websockets.serve(echo, "localhost", 8765):
```

Comment:
maybe this we can set it  to server part, it more like task need to be done in backend

Reply:
This is just a dummy server we use to make frontend API